### PR TITLE
Remove typekit to use local fonts

### PR DESCRIPTION
**Pull request by cristiangiagante**
## Background
  
Fonts are different between the new domain and Doppler Relay due to new domain is not added to Adobe TypeKit package.
  
## Changes done
  
Remove TypeKit dependency and implement local custom fonts
Add web.config
Add mime type in IIS to handle static font files
  
## Pending to be done
  
~~Be aligned with local font implementation in Doppler [(Local fonts in Doppler)](https://github.com/MakingSense/Doppler_Website/commit/51bce55b1040fed9fa9e90a44c95fc773ccb0643)~~
  
## Notes
  
~~Currently we are using only otf fonts [(Compatibility grid)](https://caniuse.com/#feat=ttf)~~
 
## Demo
 
image

image

image

### CODE REVIEW

**Comment by andresmoschini** [comment]
Nice! @mvillaseco and @fcoronel could you please take a look?

**Comment by cristiangiagante** [comment]
@andresmoschini, our local web.config will update configs on IIS only for our solution instance.

About the files, Mariana gave me this set of fonts to use, I check with web browsers compatibility and appear to be ok but I will ask again about this difference.

**Comment by andresmoschini** [comment]
> About the files, Mariana gave me this set of fonts to use, I check with web browsers compatibility and appear to be ok but I will ask again about this difference.

Please confirm before continue.

@fcoronel what do you think?

**Comment by andresmoschini** [comment]
CC: @fcoronelmakingsense

**Comment by fcoronelmakingsense** [comment]
Hi! i never use only .otf extention. Maybe need .ttf .woff for browsers compatibilty. You check is .otf works in all browsers? https://css-tricks.com/snippets/css/using-font-face/

**Comment by andresmoschini** [comment]
I cannot find my previous comment (I have to remember not commenting in commits!),

When I saw that, I realized that some font types are missing, for example `.eot`, `.ttf`, `woff`, etc. Could you research about it? see this commit: MakingSense/Doppler_Website@51bce55

**Comment by andresmoschini** [comment]
Thanks @fcoronelmakingsense !

**Comment by andresmoschini** [comment]
@mvillaseco, based in that compatibility grid, could you verify if it meets our WebApp standards?

On the other hand, @cristiangiagante, supposing that it meets our minimum standards, since making it more compatible is relatively easy: it has already done in other places (for example Doppler Site), and it does not require any research: do you think that it is a good idea resigning it taking into account cost and benefits?

**Comment by mvillaseco** [comment]
I really like this! :) only one thing, All browsers are pointing to WOFF and Woff2 in the future, maybe we should add the extension to them also(as fede link says).

**Comment by cristiangiagante** [comment]
I think it is a good idea implement all that give us more compatibility. Recently I add the others Doppler  fonts.

File: src/wwwroot/styles/core/_settings.scss [hunk 1]
```diff
@@ -29,6 +29,57 @@ $suit-font-weight-bold: 			700;
...
+$suit-font-dir: "../fonts/";
+
+@font-face {
+  font-family: $suit-custom-font-family;
+  font-style: normal;
+  font-weight: $suit-font-weight;
+  src: url($suit-font-dir + 'proxima-nova.eot');
+	src: url($suit-font-dir + 'proxima-nova.eot?#iefix') format('embedded-opentype'),
+	url($suit-font-dir + 'proxima-nova.woff2') format('woff2'),
+	url($suit-font-dir + 'proxima-nova.woff') format('woff'),
```
Comment (andresmoschini) [inline src/wwwroot/styles/core/_settings.scss]: Indentation issue?

File: src/wwwroot/styles/core/_settings.scss [hunk 2]
```diff
@@ -29,6 +29,57 @@ $suit-font-weight-bold: 			700;
...
+}
+
+@font-face {
+  font-family: $suit-custom-font-family;
+  font-style: italic;
+  font-weight: $suit-font-weight;
+  src: url($suit-font-dir + 'proxima-nova-italic.eot');
+	src: url($suit-font-dir + 'proxima-nova-italic.eot?#iefix') format('embedded-opentype'),
+	url($suit-font-dir + 'proxima-nova-italic.woff2') format('woff2'),
+	url($suit-font-dir + 'proxima-nova-italic.woff') format('woff'),
```
Comment (andresmoschini) [inline src/wwwroot/styles/core/_settings.scss]: Idem

File: src/wwwroot/styles/core/_settings.scss [hunk 3]
```diff
@@ -29,6 +29,57 @@ $suit-font-weight-bold: 			700;
...
+}
+
+@font-face {
+  font-family: $suit-custom-font-family;
+  font-style: normal;
+  font-weight: $suit-font-weight-bold;
+  src: url($suit-font-dir + 'proxima-nova-bold.eot');
+	src: url($suit-font-dir + 'proxima-nova-bold.eot?#iefix') format('embedded-opentype'),
+	url($suit-font-dir + 'proxima-nova-bold.woff2') format('woff2'),
+	url($suit-font-dir + 'proxima-nova-bold.woff') format('woff'),
```
Comment (andresmoschini) [inline src/wwwroot/styles/core/_settings.scss]: Idem

File: src/wwwroot/styles/core/_settings.scss [hunk 4]
```diff
@@ -29,6 +29,57 @@ $suit-font-weight-bold: 			700;
...
+}
+
+@font-face {
+  font-family: $suit-custom-font-family;
+  font-style: italic;
+  font-weight: $suit-font-weight-bold;
+  src: url($suit-font-dir + 'proxima-nova-bold-italic.eot');
+	src: url($suit-font-dir + 'proxima-nova-bold-italic.eot?#iefix') format('embedded-opentype'),
+	url($suit-font-dir + 'proxima-nova-bold-italic.woff2') format('woff2'),
+	url($suit-font-dir + 'proxima-nova-bold-italic.woff') format('woff'),
```
Comment (andresmoschini) [inline src/wwwroot/styles/core/_settings.scss]: Idem

File: src/wwwroot/web.config [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<configuration>
+    <system.webServer>
+        <staticContent>
+            <mimeMap fileExtension=".otf" mimeType="font/opentype" />
+            <mimeMap fileExtension=".woff" mimeType="font/woff" />
+            <mimeMap fileExtension=".woff2" mimeType="font/woff2" />
```
Comment (andresmoschini) [inline src/wwwroot/web.config]: You have missed `.ttf` and `.eot`, it seems to be an important issue (not like indentation), please fix it.

**Pull request merged**